test_server.py

```diff
@@ -34,6 +34,25 @@ def test_get_product_list(self):
         self.assertTrue( resp.status_code == HTTP_200_OK )
         self.assertTrue( len(resp.data) > 0 ) 
 
+    def test_get_product_price(self):
```

Comment:
As mentioned in https://github.com/NYU-DevOps-Charlie-CSCI-GA-3033-013/products/pull/23, I believe the `test_get_product_price(self)` method should perform three GET requests:

1. `GET /products?min=100`: Test the lower boundary -> should return only 'TV' for $999
2. `GET /products?max=100`: Test the upper boundary -> should return only 'Blender' for $99
3. `GET /products?min=100&max=1000`: Test both boundaries working together as a range -> should return only 'TV' for $999. I'm not sure if this third case is necessary, though. Ideally we don't want to clutter our unit test class with unnecessary methods that test functionality that has already been covered by previous steps.